src/pages-and-resources/teams/Settings.jsx

```diff
@@ -0,0 +1,82 @@
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { Button, TransitionReplace } from '@edx/paragon';
+import { faPlus } from '@fortawesome/free-solid-svg-icons';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
use icon from paragon instead of FontAwesomeIcon, it will also remove the dependency of font awesome library. 
`import {Add} from '@edx/paragon/icons';`
for reference https://paragon-edx.netlify.app/foundations/icons

Reply:
Sure, will do. 

Unfortunately, the selection of icons in paragon is currently limited, so we'll still need to use it in Discussions till more icons are added.

Reply:
we can make a request for missing icons to be added in paragon. the currently required icons are available in paragon.